lambda_services/job_service/launcher/weboptions.py

```diff
@@ -146,7 +142,7 @@ def __init__(self, form):
         # if form.has_key("LIGAND") and form['LIGAND'].filename:
         # self.ligandfilename=sanitizeFileName(form["LIGAND"].filename)
         if "LIGANDFILE" in form and form["LIGANDFILE"] != "":
-            self.ligandfilename = sanitize_fileName(form["LIGANDFILE"])
+            self.ligandfilename = sanitize_file_name(form["LIGANDFILE"])
```

Comment:
Ran into an exception here when running in Test.  Looks like we forgot a `self.job_tag` parameter before `form["LIGANDFILE"]`